src/encoded/static/components/search.js

```diff
@@ -405,6 +406,14 @@ const ExperimentComponent = (props, reactContext) => {
         )).map((replicate) => replicate.library.biosample.subcellular_fraction_term_name));
     }
 
+    // Collect examined loci
+    if (result.examined_loci && result.examined_loci.length > 0) {
+        result.examined_loci.forEach((loci) => {
+            examinedLoci.push(loci.gene.symbol);
+        });
+        examinedLoci = _.uniq(examinedLoci);
```

Comment:
This is code. But personally, I prefer to use third party libraries only when there is no in-built substitute. You could use Set.